XNAControls/XNAMultiColumnListBox.cs

```diff
@@ -48,8 +48,15 @@ private void XNAMultiColumnListBox_ClientRectangleUpdated(object sender, EventAr
         lastHeader.Width = lastColumnWidth;
     }
 
+    public delegate void HoveredIndexChangedEventHandler(object sender, EventArgs e);
+    public event HoveredIndexChangedEventHandler HoveredIndexChanged;
+
     public delegate void SelectedIndexChangedEventHandler(object sender, EventArgs e);
     public event SelectedIndexChangedEventHandler SelectedIndexChanged;
+
+    public delegate void TopIndexChangedEventHandler(object sender, EventArgs e);
+    public event TopIndexChangedEventHandler TopIndexChanged;
```

Comment:
Use the built-in C# EventHandler class.

```suggestion
    public event EventHandler TopIndexChanged;
```

Change the others similarly.

Yeah, I know `SelectedIndexChanged` uses a custom event handler, but that's just because it's old and I haven't got around to refactoring all such cases.